libipl/libipl.H

```diff
@@ -40,6 +40,7 @@ enum ipl_error_type {
 	IPL_ERR_SPARE_CLK,
```

Comment:
**Commit message:**

- Can you please append the prefix `libipl: p10:`?

Reply:
Sure